purchases-js-hybrid-mappings/src/purchases-common.ts

```diff
@@ -104,6 +104,15 @@ export class PurchasesCommon {
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  static overridePreferredLocale(_locale: string | null): void {
```

Comment:
Hmm do we need this method at all? If it's to just make it fall, we can configure that on each hybrid (at least, that's what we were doing for Flutter + react native. I would probably just remove all changes in `purchases-js-hybrid-mappings` but leave the ones in `purchases-typescript-internal`